warrior/Framework/Utils/encryption_utils.py

```diff
@@ -35,6 +35,18 @@
 
```

Comment:
except ImportError:
   pNote ("Please Install Pycryptodome 3.6.1 and above", "error")

We need to recommend a supported version and declare it as the recommended one.

Reply:
Done. Please suggest any other messages.